pkg/proc/eval.go

```diff
@@ -118,6 +118,13 @@ func (scope *EvalScope) Locals() ([]*Variable, error) {
 			addr := afterLastArgAddr(vars)
 			if addr == 0 {
 				addr = uintptr(scope.Regs.CFA)
+				// The address needed on the ARM64 by the Dwarf lookup routines
```

Comment:
If CFA was wrong then a lot of things wouldn't work so the suggestion that it should be changed is inappropriate.

Reply:
I was intending the comment only as a possible reminder to look at all usages of CFA at some time in the future.  It seems odd to use CFA on the AMD64 architecture but to use CFA + 8 on the AMD64.  I'm fine with the comment being removed.

Reply:
This piece of code is used to work around a bug in the compiler where sometimes the arguments of optimized functions do not get an address recorded in debug_info. It almost never runs. Conversely the value of CFA is used to calculate the value of every other variable as well as implementing step over. If it was wrong almost nothing would work.